baseline.yaml

```
@@ -196,18 +196,24 @@ criteria:
     maturity_level: 1
     category: Build & Release
     criteria: |
-      The project's build and release pipelines
-      MUST NOT execute arbitrary code that is
-      input from outside of the build script.
+      Build and release pipelines MUST only execute
+      code that's present in the repository *or* from external trusted sources.
```

```suggestion
      code that's either present in the repository *or* from external sources trusted by the project.
```

The text said "trusted" but not "trusted by whom?" We need to make it clear that it's the *project* that determines what external sources are trusted. I put the "either" there to make sure it's clear the *or* separates two clauses.